src/components/RadioInputGroup/RadioInput/RadioInput.stories.tsx

```diff
@@ -0,0 +1,53 @@
+import { useEffect, useState } from 'react';
+import { ComponentMeta, ComponentStory } from '@storybook/react';
+import { RadioInput, RadioInputProps } from './RadioInput';
+import { WithGrayBackground } from '../../../hoc/WithGrayBackground';
+
+export default {
+  title: 'Components/RadioButtonGroup/RadioButton',
+  component: RadioInput,
+} as ComponentMeta<typeof RadioInput>;
+
+const Template: ComponentStory<typeof RadioInput> = ({
+  isChecked: initialIsChecked,
+  onChange,
+  ...remainingProps
+}: RadioInputProps) => {
+  const [isChecked, setIsChecked] = useState(initialIsChecked);
+
+  useEffect(() => {
+    setIsChecked(initialIsChecked);
+  }, [initialIsChecked]);
+
+  return (
+    <WithGrayBackground>
+      <RadioInput
+        {...remainingProps}
+        onChange={(id, value) => {
+          setIsChecked(!isChecked);
+          onChange(id, value);
+        }}
```

Comment:
Why do we have `onChange` twice here?

Reply:
We provide an arrow function to the `RadioInput` `onChange` property that internally calls the received `onChange` callback.

To a function we provide a function that calls another function with the same name.
This needed to be done due to story specifics (the need to control a component both by logic and by controls)